src/import-mongo.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import mongo from 'mongodb';
+import BulkBadger from 'bulkbadger';
+import { getUrlFromCluster } from './utils';
+
+export function cli (cluster, database,  mongourl, collection) {
+  if (!cluster || !database || !mongourl || !collection) {
```

Comment:
can we also test for a valid mongodb url, and add that to EUSAGE, so the driver does not bail with a long stacktrace if i forget it?

```
(10:40:30) [robert@tequila-work] ~/apache/couchdb-nmo (pr/16) $ ./bin/nmo-cli.js import-mongo anemone restaurants foo restaurants
ERR! invalid schema, expected mongodb
ERR! Error: invalid schema, expected mongodb
ERR!     at module.exports (/Users/robert/apache/couchdb-nmo/node_modules/mongodb/lib/url_parser.js:20:11)
ERR!     at connect (/Users/robert/apache/couchdb-nmo/node_modules/mongodb/lib/mongo_client.js:125:16)
ERR!     at Function.MongoClient.connect (/Users/robert/apache/couchdb-nmo/node_modules/mongodb/lib/mongo_client.js:109:3)
```